pkg/common/pod_config.go

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package common
+
+import (
+	"context"
+
+	internalapi "k8s.io/cri-api/pkg/apis"
+	runtimev1 "k8s.io/cri-api/pkg/apis/runtime/v1"
+)
+
+func GetCgroupParent(ctx context.Context, c internalapi.RuntimeService) string {
+	runtimeConfig, err := c.RuntimeConfig(ctx)
+	if err != nil {
+		return ""
+	}
+	if runtimeConfig == nil || runtimeConfig.Linux == nil {
+		return ""
+	}
+	cgroupDriver := runtimeConfig.Linux.GetCgroupDriver()
+	if cgroupDriver == runtimev1.CgroupDriver_CGROUPFS {
+		return ""
+	}
+	return "/test.slice"
```

Comment:
Is this a randomly chosen slice name?

Reply:
`test` is my randomly chosen,  `.slice` is must this.

Reply:
an option is we could use the same scheme as the kubelet, like `kubepods-pod$POD_UID.slice`